docs/source/automated_feature_engineering/handling_time.rst

```diff
@@ -131,6 +131,53 @@ For example, a customer's session has multiple transactions which can happen at
 
 Featuretools can automatically add last time indexes to every :class:`Entity` in an :class:`Entityset` by running ``EntitySet.add_last_time_indexes()``. If a ``last_time_index`` has been set, Featuretools will check to see if the ``last_time_index`` is after the start of the training window. That, combined with the cutoff time, allows DFS to discover which data is relevant for a given training window.
 
+
+Excluding data at cutoff times
+-----------------------------------------------
+
+If you don't want to use the data at cutoff times in feature calculation, you can exclude them by setting ``include_cutoff_time`` to ``False``. Then, it will not use the data from these time points in time or any data newer than that. If you set it to ``True``, it will use the data from the older(or within the ``training_window`` if you set) and the one from the cutoff time as well:
```

Comment:
How does this sound?

"The `cutoff_time` is the last point in time where data can be used for feature calculation., but you may want to specify the first time point that cannot be used instead.  If you don't want to use the data at the cutoff time in feature calculation, you can exclude that data by setting ``include_cutoff_time`` to ``False`` in `dfs` or `calculate_feature_matrix`.  If you set it to ``True`` (the default behavior), data from the cutoff time point will be used."

I do think there should be a second paragraph on how training window works (or maybe it can be part of the examples).  Since with `include_cutoff_time=True`, the oldest point in the training window is excluded an the cutoff time point is included.  Whereas with `include_cutoff_time=False` the oldest point the window is included and the cutoff time point is excluded.  I think the example should demonstrate this as well.  If the training window was 130 seconds long transaction 308 would be on the oldest point on the window to demonstrate that behavior

If you want to have regular text in between the ipython cells you can break the code into multiple ipython blocks.  They'll still get run in order.

Reply:
Thanks for these updates 😄